document/core/exec/modules.rst

```diff
@@ -371,6 +371,66 @@ New instances of :ref:`functions <syntax-funcinst>`, :ref:`tables <syntax-tablei
    \end{array}
 
 
+.. index:: element, element instance, element address
+.. _alloc-elem:
+
+:ref:`Element segments <syntax-eleminst>`
+.........................................
+
+1. Let :math:`e` be the :ref:`element segment <syntax-elem>` to allocate.
+
+2. If :math:`e` is of the form :math:`\{ \EINIT~x^\ast \}`, then:
```

Comment:
What does this algorithm return otherwise? Should it be epsilon, i.e., the algo returns an optional address?